src/lib/1_Atoms/Icon.svelte

```diff
@@ -1,13 +1,13 @@
 <script>
-  export let name = ''
+  export let name = '';
 </script>
 
 {#if name == 'bedrijf'}

```

Comment:
We moeten alleen nog kijken om deze te veranderen en meer variabele in het project